code/datums/emergency_calls/emergency_call.dm

```diff
@@ -91,7 +91,7 @@
 
 	for(var/mob/dead/observer/M in GLOB.observer_list)
 		if(M.client)
-			to_chat(M, FONT_SIZE_LARGE("\n<span class='attack'>[ert_message]. >>> <a href='?src=\ref[M];joinresponseteam=1;'><b>Join Response Team</b></a></span> <<<"))
+			to_chat(M, FONT_SIZE_LARGE("\n<span class='attack'>[ert_message]. >>> <a href='?src=\ref[M];joinresponseteam=1;'><b>Join Response Team</b></a> <<<</span>"))
```

Comment:
you should not be putting in >>> and <<< in html statements, these should be &lt; and &gt;

Reply:
sorry this was a mistake because html memes, you need to use `&lt;` and `&gt;`